README.md

```diff
@@ -9,27 +9,34 @@
 
 <a href="https://info.blackducksoftware.com/OpenSourceRookies2015.html" ><img src="/media/images/Rookies_Award_Badge.png" width="150" alt="Hygieia℠ is now BlackDuck 2015 OpenSource Rookie of the year"></a>
 
-### What is Hygieia℠
+### What is Hygieia℠?
 
 ##### Pronunciation:      hi-gee-ya (origin Greek)
-Hygieia℠ is a single, configurable, easy to use dashboard to visualize near real-time status of the entire delivery pipeline.
+Hygieia℠ is a single, configurable, easy-to-use dashboard to visualize near real-time status of the entire delivery pipeline.
 
-Please view a video below to see Hygieia℠ in action
+Watch the video below to see Hygieia℠ in action:
 
  [![Hygieia℠ Is AWESOME](/media/images/video-shot.png)](https://www.youtube.com/watch?v=SoNTA78j0tc "Hygieia Is AWESOME")
 
-We use tools for everything today and each tool has its own unique dashboard to present information. What is lacking is a near real time consolidated view of the end-to-end delivery pipeline.  This view and a continuous feedback loop are critical to the success of
-any DevOps organization.
+We use tools for everything today, and each tool has its own unique dashboard to present information. What is lacking is a near real-time consolidated view of the end-to-end delivery pipeline. This view and a continuous feedback loop are critical to the success of any DevOps organization.
 
-Capital One Engineers from our DevOps team developed Hygieia keeping in mind the need for such a tool since none is available commercially out of the box in ready to use the state as well as ease of use.
-For the past two years, we’ve looked for DevOps visualization tools in the commercial market as well as in the Open Source community
```

Comment:
Updated to:

`Capital One Engineers from our DevOps team developed Hygieia with the need for a user-friendly, out-of-the-box DevOps visualization tool in mind since none is available commercially. We researched existing tools and found:`

Hopefully this works for you! Thanks for your comments... I am learning how to use GitHub and this has been super helpful.